spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -47,19 +47,147 @@ definitions:
      - discontinuity_counter:
             type: u8
             desc: |
-              Signal phase discontinuity counter. 
+              Signal phase discontinuity counter.
               Increased for every discontinuity in phase.
      - bias:
             type: s32
             units: 0.1 mm
             desc: Phase bias for specified signal
 
+ - STECHeader:
+    short_desc: Header for MSG_SSR_STEC_CORRECTION message
+    desc: |
+      A full set of STEC information will likely span multiple SBP
+      messages, since SBP message a limited to 255 bytes.  The header
+      is used to tie multiple SBP messages into a sequence.
+    fields:
+    - time:
+        type: GPSTime
+        desc: GNSS time of the STEC data
+    - num_msgs:
+        type: u8
+        desc: Number of messages in the dataset
+    - seq_num:
+        type: u8
+        desc: Position of this message in the dataset
+    - ssr_update_interval:
+        type: u16
+        desc: update interval in seconds
+    - iod_ssr:
+        type: u8
+        desc: range 0 - 15
+
+ - GriddedCorrectionHeader:
+    short_desc: Header for MSG_SSR_GRIDDED_CORRECTION
+    desc: |
+      The 3GPP message contains nested variable length arrays
+      which are not suppported in SBP, so each grid point will
+      be identified by the index.
+    fields:
+    - time:
+        type: GPSTime
+        desc: GNSS time of the STEC data
+    - num_msgs:
+        type: u16
+        desc: Number of messages in the dataset
+    - seq_num:
+        type: u16
+        desc: Position of this message in the dataset
+    - ssr_update_interval:
+        type: u16
+        desc: update interval in seconds
+    - iod_ssr:
+        type: u8
+        desc: range 0 - 15
+    - tropo_quality:
+        type: u8
+        desc: troposphere quality indicator
+
+ - STECSatElement:
+     desc: STEC for the given satellite.
+     fields:
+     - sv_id:
+         type: SvId
+         desc: Unique space vehicle identifier
+     - stec_quality_indicator:
+         type: u8
+         desc: quality of STEC data
+     - stec_coeff:
+         type: array
+         size: 4
+         fill: s16
+         desc: coefficents of the STEC polynomial
+
+ - TroposphericDelayCorrection:
+    short_desc: troposphere delay correction
+    desc: |
+      Contains wet vertical and hydrostatic vertical delay
+    fields:
+    - hydro:
+        type: s16
+        desc: hydrostatic vertical delay
+    - wet:
+        type: s8
+        desc: wet vertical delay
+
+ - STECResidual:
+    desc: STEC residual
+    fields:
+    - sv_id:
+        type: SvId
+        desc: space vehicle identifier
+    - residual:
+        type: s16
+        desc: STEC residual (Scale factor 0.04 TECU)
```

Comment:
and what's "U" here?  Total Electron Content...? @benjamin0

Reply:
Unit